src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -1482,6 +1486,8 @@ public static String getBasesInWindowAroundReferenceAllele( final Allele refAlle
                                                                 final ReferenceContext referenceContext) {
         // TODO: This is generating too long a string for INDELs because of VCF format alleles - see Issue #4407
 
+        // TODO: this seems to be the same as GencodeFunotationFactory::getReferenceBases - should that method call into this?
```

Comment:
Can we determine an answer and apply a fix?  Otherwise, file an issue and reference it here.

Reply:
This is the "ugliness" I mentioned to you in person.  My plan is to file an issue and keep this in for now so the fix can go in sooner.